### DISCUSSION TITLE
unhashable type: 'dict' - error while "poetry add ..." New

### DISCUSSION THREAD

**Discussion by tsytsyksvitlana** [python-poetry/poetry#9804] [Q&A]
I get this error when running poetry install, poetry add, poetry remove

I tried to create a new venv on Python 3.13, 3.12 I used poetry 1.8.4, 1.8.0 Empty projects created with poetry initialize the same when trying to add In all the code, in paycharm, in the terminal too I also completely uninstalled and installed python and poetry

**Comment by finswimmer** [comment]
Hey @tsytsyksvitlana,

please include the complete output you receive on `poetry add`. Also please provide the `pyproject.toml` of the project.

fin swimmer
